src/metaTypes.h

```diff
@@ -35,18 +36,18 @@ namespace METAIO_NAMESPACE {
 typedef char                MET_ASCII_CHAR_TYPE;
 typedef char                MET_CHAR_TYPE;
```

Comment:
I believe the intention of the "MET_CHAR_TYPE" is to be signed. The "char" type could be either singed or unsigned. I believe that "int8_t" is needed to make it clear how to interpret this type.